src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -148,8 +167,9 @@ private static LocalDateTime getStartOfDate(LocalDateTime current, ZoneId zoneId
 
     /**
      * Get the earliest commit date from {@code commitInfos}.
-     * @return First commit date if there is at least one {@code CommitResult}. Otherwise, return
-     * the {@code ARBITRARY_FIRST_COMMIT_DATE} converted to the timezone given by {@code zoneId}.
+     *
+     * @return First commit date if there is at least one {@link CommitResult}. Otherwise, return
+     * the {@code ARBITRARY_FIRST_COMMIT_DATE_UTC} converted to the timezone given by {@code zoneId}.
```

Comment:
should be `{@link Class#ARBITRARY_FIRST_COMMIT_DATE_UTC}`?

Reply:
This shows a non-working red link. I also realised that ARBITRARY_FIRST_COMMIT_DATE_UTC is a private constant, so it may not be worth mentioning directly.